### ISSUE TITLE
Usage

### ISSUE THREAD

**Issue by wsjackson14** [Emergen-SEE/Emergen-SEE-Situations#14]
I like the idea of your product, however I find myself wondering how widely used it would really be? For example, for at risk patients or elderly people, they have life alert, which is basically another version of your product. And then in the case of someone being in danger, why would they need to get on their phone and go to the app when they could just call 911 to ensure the situation is responded to correctly? Another question I have is how would you go about partnering up with first responders and medical professionals and also with the phone companies?

**Comment by daairola** [comment]
I was thinking the same thing. I don't understand how the product will be unique compared to other products we've seen before, or how it would solve an issue that we don't already have a solution for.

**Comment by ChristinaNguyen9** [comment]
I agree with the both of you, there's already so many companies in the market that have the same concept.

**Comment by lmmuellner** [comment]
I agree with everyone here, how might your product be more unique in order to gain attention from customers. That might be using competitors products.
